packages/yoroi-extension/app/components/wallet/navigation/WalletNavigation.js

```diff
@@ -38,7 +39,7 @@ export default class WalletNavigation extends Component<Props> {
         <WalletNavButton
           className={category.className}
           label={intl.formatMessage(category.label)}
-          icon={category.icon}
+          icon={category.label.id === WALLET_TRANSACTION_ID && category.isActive ? TransactionImageActive  : category.icon}
```

Comment:
@AhmedIbrahim336 , we have multiple tabs in the interface and all of them have icons that change color, but other tabs don't declare a check like this here. Seems like there must be another solution for it.

Also I see that this code will use the new icon when the tab is active, but when the tab is not active - we will see the old icon. The task is not resolved.